packages/ssr-simulator/config/webpackConfigFactory.js

```diff
@@ -139,22 +139,22 @@ module.exports = function(envType) {
     ].filter(Boolean)
   };
 
-  config.optimization = IS_DEV
-    ? {}
-    : {
-        minimizer: [
-          new UglifyJsPlugin({
-            parallel: true,
-            sourceMap: true,
-            uglifyOptions: {
-              output: {
-                comments: false
-              }
-            }
-          }),
-          new OptimizeCSSAssetsPlugin({})
-        ]
-      };
+  // config.optimization = IS_DEV
+  //   ? {}
+  //   : {
+  //       minimizer: [
+  //         new UglifyJsPlugin({
+  //           parallel: true,
```

Comment:
@guysopher Turns out `uglifyJS` doesn't support ES6. Since it's an optimisation for a playground project I disabled it's usage. FYI